readme.md

```diff
@@ -711,6 +711,51 @@ Config Based Styles Can be Applied to Components.
     ]
 }
 
+### Query Param Support With React Crux
+We can give queryParams in config. It will append queryparams with every fetch call.
+
+```
+import * as React from "react"
+import { CruxComponentCreator } from "@curefit/react-crux"
+
+const constants = {
+    modelName: "serviceAccess",
+    title: "Service Access",
+    creationTitle: "Service Access",
+    createModal: true,
+    editModal: true,
+    largeEdit: true,
+    stateRoot: "none",
+    fields: [
+        {
+            title: "Service",
+            field: "serviceName",
+            representative: true,
+        },
+        {
+            title: "Users",
+            field: "users",
+            type: "iterable",
+            iterabletype: {
+                title: "User",
+                inlineEdit: true
+            }
+        }
+    ]
+}
+
+const ServiceAccessComponent = CruxComponentCreator.create<ServiceAccess, ServiceAccessProps>(constants)
+
+class ComponentWithQueryParams extends React.Component<{}, {}> {
```

Comment:
I am not getting the example. How are the two components related ?

Reply:
ServiceAccessComponent is created with react-crux. While rendering that component in another component. we can pass props(queryParams) to it. 
these query params will be appended in every service call from react-crux

Reply:
OK. Earlier some other component was beigng used here called "Forms".
Anyways ... instead of passing a prop called queryParams, you should pass prop called options and queryParams should be a field inside that object